### PR TITLE
Static_pages

### PR DESCRIPTION
**Pull request by hienbx94**
I have created new pull request, check it please

### CODE REVIEW

File: app/assets/javascripts/static_pages.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```
Comment (phamkykhoi) [inline app/assets/javascripts/static_pages.coffee]: Use git rm take remove this file

File: app/assets/stylesheets/static_pages.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the StaticPages controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (phamkykhoi) [inline app/assets/stylesheets/static_pages.scss]: Use git rm take remove this file

File: app/helpers/static_pages_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
+end
```
Comment (phamkykhoi) [inline app/helpers/static_pages_helper.rb]: Use git rm take remove this file if not use

File: app/views/layouts/_footer.html.erb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<footer class="footer">
+  <small>
+    The <a href="#">Framgia E-Learning</a>
```
Comment (phamkykhoi) [inline app/views/layouts/_footer.html.erb]: I18n

File: app/views/layouts/_footer.html.erb [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+    by <a href="#">Framgia</a>
```
Comment (phamkykhoi) [inline app/views/layouts/_footer.html.erb]: I18n

File: app/views/layouts/_footer.html.erb (hunk 4 above)
Comment (phamkykhoi) [inline app/views/layouts/_footer.html.erb]: Not use tag a. Should use link_to

File: app/views/layouts/_footer.html.erb (hunk 5 above)
Comment (phamkykhoi) [inline app/views/layouts/_footer.html.erb]: Not use tag a. Should use link_to

File: app/views/layouts/_footer.html.erb [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+  </small>
+  <nav>
+    <ul>
+      <li><%= link_to "About",   about_path %></li>
+      <li><%= link_to "Contact", contact_path %></li>
+      <li><a href="http://news.railstutorial.org/">News</a></li>
```
Comment (phamkykhoi) [inline app/views/layouts/_footer.html.erb]: Not use tag a. Should use link_to

File: app/views/layouts/_header.html.erb [hunk 7]
```diff
@@ -0,0 +1,12 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to "Framgia E-Learning", root_path, id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to "Home",    root_path %></li>
```
Comment (phamkykhoi) [inline app/views/layouts/_header.html.erb]: I18n

File: app/views/layouts/_header.html.erb [hunk 8]
```diff
@@ -0,0 +1,12 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to "Framgia E-Learning", root_path, id: "logo" %>
```
Comment (phamkykhoi) [inline app/views/layouts/_header.html.erb]: I18n

File: app/views/layouts/_header.html.erb [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to "Home",    root_path %></li>
+        <li><%= link_to "Help",    help_path %></li>
+        <li><%= link_to "Log in", '#' %></li>
```
Comment (phamkykhoi) [inline app/views/layouts/_header.html.erb]: I18n

File: app/views/layouts/_header.html.erb [hunk 10]
```diff
@@ -0,0 +1,12 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to "Framgia E-Learning", root_path, id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to "Home",    root_path %></li>
+        <li><%= link_to "Help",    help_path %></li>
```
Comment (phamkykhoi) [inline app/views/layouts/_header.html.erb]: I18n

File: app/views/layouts/_header.html.erb (hunk 9 above)
Comment (phamkykhoi) [inline app/views/layouts/_header.html.erb]: ' ' => " "

File: app/views/static_pages/about.html.erb [hunk 11]
```diff
@@ -0,0 +1,2 @@
+<% provide(:title, "About") %>
```
Comment (phamkykhoi) [inline app/views/static_pages/about.html.erb]: I18n

File: app/views/static_pages/help.html.erb [hunk 12]
```diff
@@ -0,0 +1,4 @@
+<% provide(:title, "Help") %>
```
Comment (phamkykhoi) [inline app/views/static_pages/help.html.erb]: I18n

File: app/views/static_pages/home.html.erb [hunk 13]
```diff
@@ -0,0 +1,10 @@
+<div class="center jumbotron">
+  <h1> Framgia E-Learning </h1>
```
Comment (phamkykhoi) [inline app/views/static_pages/home.html.erb]: I18n

File: app/views/static_pages/home.html.erb [hunk 14]
```diff
@@ -0,0 +1,10 @@
...
+
+  <h2>
+    This is the home page for the Framgia E-Learning System
```
Comment (phamkykhoi) [inline app/views/static_pages/home.html.erb]: I18n

File: app/views/static_pages/home.html.erb [hunk 15]
```diff
@@ -0,0 +1,10 @@
+<div class="center jumbotron">
+  <h1> Framgia E-Learning </h1>
+
```
Comment (phamkykhoi) [inline app/views/static_pages/home.html.erb]: Remove this line empty

File: app/views/static_pages/home.html.erb [hunk 16]
```diff
@@ -0,0 +1,10 @@
...
+  <h2>
+    This is the home page for the Framgia E-Learning System
+  </h2>
+
+  <%= link_to "Sign up now!", '#', class: "btn btn-lg btn-primary" %>
```
Comment (phamkykhoi) [inline app/views/static_pages/home.html.erb]: I18n

Comment (phamkykhoi) [inline app/views/static_pages/home.html.erb]: ' ' => " "

File: app/helpers/application_helper.rb [hunk 17]
```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
-end
+  # Returns the full title on a per-page basis.
+  def full_title(page_title = "")
+    base_title = "Framgia E-Learning"
```
Comment (phamkykhoi) [inline app/helpers/application_helper.rb]: I18n

File: app/views/layouts/application.html.erb [hunk 18]
```diff
@@ -1,14 +1,18 @@
...
-<body>
-
-<%= yield %>
-
-</body>
-</html>
+  <head>
+    <title><%= full_title(yield(:title)) %></title>
+    <%= stylesheet_link_tag "application", media: "all",
+                                           "data-turbolinks-track" => true %>
```
Comment (phamkykhoi) [inline app/views/layouts/application.html.erb]: not use =>
use :

File: app/views/layouts/application.html.erb [hunk 19]
```diff
@@ -1,14 +1,18 @@
...
+    <%= javascript_include_tag "application", "data-turbolinks-track" => true %>
```
Comment (phamkykhoi) [inline app/views/layouts/application.html.erb]: not use =>
use :

File: app/helpers/application_helper.rb [hunk 20]
```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
-end
+  # Returns the full title on a per-page basis.
+  def full_title(page_title = "")
+    base_title = I18n.t("framgia")
+    if page_title.empty?
```
Comment (phamkykhoi) [inline app/helpers/application_helper.rb]: Should use age_title.empty? ? base_title : page_title + " | " + base_title

File: app/helpers/application_helper.rb [hunk 21]
```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
-end
+  # Returns the full title on a per-page basis.
```
Comment (dieunb) [inline app/helpers/application_helper.rb]: remove all remark

File: app/helpers/application_helper.rb [hunk 22]
```diff
@@ -1,2 +1,7 @@
...
+  def full_title(page_title = "")
```
Comment (dieunb) [inline app/helpers/application_helper.rb]: remove `()`

File: app/views/layouts/application.html.erb [hunk 23]
```diff
@@ -1,14 +1,17 @@
...
-  <%= csrf_meta_tags %>
-</head>
-<body>
-
-<%= yield %>
-
-</body>
-</html>
+  <head>
+    <title><%= full_title(yield(:title)) %></title>
```
Comment (dieunb) [inline app/views/layouts/application.html.erb]: remove `()` first

File: app/views/static_pages/about.html.erb [hunk 24]
```diff
@@ -0,0 +1,2 @@
+<% provide(:title, t("about")) %>
```
Comment (dieunb) [inline app/views/static_pages/about.html.erb]: remove `()`

File: app/views/static_pages/contact.html.erb [hunk 25]
```diff
@@ -0,0 +1,2 @@
+<% provide(:title, t("contact")) %>
```
Comment (dieunb) [inline app/views/static_pages/contact.html.erb]: remove `()`

File: app/views/static_pages/help.html.erb [hunk 26]
```diff
@@ -0,0 +1,2 @@
+<% provide(:title, t("help")) %>
```
Comment (dieunb) [inline app/views/static_pages/help.html.erb]: remove `()`

**Pull request merged**